packages/react/src/lib/accessibility/FocusZone/FocusTrapZone.tsx

```diff
@@ -57,46 +53,38 @@ export class FocusTrapZone extends React.Component<FocusTrapZoneProps, {}> {
 
   public componentDidMount(): void {
     FocusTrapZone._focusStack.push(this)
-    const { disableFirstFocus = false } = this.props
 
     this.windowElement = getWindow(this._root.current)
     this._previouslyFocusedElementOutsideTrapZone = this._getPreviouslyFocusedElementOutsideTrapZone()
 
-    if (
-      !this._root.current.contains(this._previouslyFocusedElementOutsideTrapZone) &&
-      !disableFirstFocus
-    ) {
-      this._findElementAndFocusAsync()
-    }
-
     this._hideContentFromAccessibilityTree()
     this._subscribeToEvents()
   }
 
   public render(): JSX.Element {
-    const { className, ariaLabelledBy } = this.props
+    const {
+      as,
```

Comment:
It seems that `as` is unhandled prop in this case and can be removed from `propTypes`